tests/components/doorbird/test_config_flow.py

```diff
@@ -288,6 +302,43 @@ async def test_form_zeroconf_correct_oui(hass):
     assert len(mock_setup_entry.mock_calls) == 1
 
 
+@pytest.mark.parametrize(
+    "doorbell_state_side_effect",
+    [
+        requests.exceptions.HTTPError(response=Mock(status_code=404)),
+        OSError,
+        None,
+    ],
+)
+async def test_form_zeroconf_correct_oui_wrong_device(hass, doorbell_state_side_effect):
+    """Test we can setup from zeroconf with the correct OUI source but not a doorstation."""
+    await hass.async_add_executor_job(
+        init_recorder_component, hass
```

Comment:
Do you know why we need to set up the recorder? I don't remember the reason.

Reply:
I can't remember the exact reason but I do remember the tests would fail the CI without it 

Let's do an experiment and see if we can take it out  https://github.com/home-assistant/core/pull/48781